### PR TITLE
Joakim's and Pilars library challenge

### PR DESCRIPTION
**Pull request by Joberg95**
PR

### CODE REVIEW

**Comment by Joberg95** [comment]
Trying to set up tests for a return date so we know if a book is available. Also updating progress in README.md as we go.

File: lib/data.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (faraznaeem) [inline lib/data.yml:1]: Why do you have two different yml files with books?

File: lib/library.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class Library
+    attr_accessor :books
+
+collection = YAML.load_file "books.yml"
+subject = data["Book_list"]
+
+def initialize
+    @library_account = active
+    @title = title #not sure how to link to yaml file
```
Comment (faraznaeem) [inline lib/library.rb:9]: Delete comments in the future before submitting the code

File: spec/borrower_spec.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+require './lib/library_account.rb'
+require 'date'
+require 'books.yml'
+require '/lib/borrower.rb'
+
+describe Borrower do
+
+    subject {described_class.new(name: 'Christina')}
+
+    He or she 'is expected to have a :name on initialize' do
```
Comment (faraznaeem) [inline spec/borrower_spec.rb:10]: this is probably not working, that is beacuse of the syntax error. Remember to have proper `it` blocks

File: spec/library_account_spec.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+require './lib/library_account.rb'
+require 'date'
+require 'books.yml'
+
+describe Library_account do
+    let(:account) {instance_double('Person', name: 'Christina'), account_status: 'active'}
+    subject {described_class.new({borrower: person})}
+    #not sure but I think we need to set up user accounts for borrowers? plus are there extra brackets above?
```
Comment (faraznaeem) [inline spec/library_account_spec.rb:8]: Comments in code is a no-go

**Review by faraznaeem — CHANGES_REQUESTED** [review]
You have done a good attempt, but the code isn't working for small reasons as syntax errors. Make sure to put more attention on the small details in the future.

**Pull request closed without merging**